__tests__/index.test.ts

```diff
@@ -155,7 +154,7 @@ describe('Auth0', () => {
   });
 
   afterEach(() => {
-    jest.resetAllMocks();
+    jest.clearAllMocks();
```

Comment:
`resetAllMocks` destroys the `localStorage` mock (which browser-tabs-lock uses in these tests) https://github.com/clarkbw/jest-localstorage-mock/issues/83

Had to fix some leaky tests after removing it